### PR TITLE
containers/project.py fix

### PR DESCRIPTION
**Pull request by jdemon519**
Fixed cfme/containers/project.py

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit faa3e24ce76d4e7ace3cfd24d77694f167ec6a48:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/containers/test_relationships_tables.py [hunk 1]
```diff
@@ -26,10 +26,10 @@
 @pytest.mark.parametrize('rel',
                          ['Containers Provider',
                           'Project',
-                          'Replicator',
-                          'Node',
                           'Services',
-                          'Containers'])
+                          'Replicator',
+                          'Containers',
+                          'Node'])
```
Comment (psav) [inline cfme/tests/containers/test_relationships_tables.py]: Any reason to reorder these?

File: cfme/tests/containers/test_views_objects.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.5"),
+    pytest.mark.usefixtures('setup_provider')]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+
+def test_pods_views():
```
Comment (psav) [inline cfme/tests/containers/test_views_objects.py]: How about a little dry here?

```python
def view_testing(location):
    views = ['Grid', 'Tile', 'List]
    for view in views:
        view_name = '{} View'.format(view)
        tb.select(view_name)
        assert tb.is_active(view_name), '{} {} setting failed'.format(location, view_name)

def test_replicators_views():
    sel.force_navigate('containers_replicators')
    view_testing('containers_replicators')
```

**Comment by psav** [comment]
We'll also want to squash these changes - did you also investigate failures?

**Comment by psav** [comment]
As #3102 contains these commits too, can we close off this PR?

**Comment by psav** [comment]
Closing on request

**Pull request closed without merging**